### ISSUE TITLE
Smooth lines on Area Charts

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#1642]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `None` on 1970-01-01T00:00:00.00Z

**Comment by antitheos** [comment]
We are building a project reporting app using google charts and this is a feature we very much need.

**Comment by AndyGaskell** [comment]
I'd be really keen to see this feature.

The smoothed lines on line charts using the "curveType" param are really nice, but it'd be good to see this extended to area charts.  We have a dashboard screen with both line and area charts and it is visually jarring that line charts have smooth lines and area charts do not.

Just a visual thing I guess, but it'd be really nice.

**Comment by jonkerr** [comment]
Agreed with all the above.  Area chart is not very nice to look out without the smoothing feature. 

Please add the curveType option to area charts.

**Comment by dlaliberte** [comment]
This should be doable, but we have other priorities.  If you want to work at it a bit, you can get a smoothed area chart by using 'area' intervals:  https://developers.google.com/chart/interactive/docs/gallery/intervals#area-intervals

**Comment by HarelM** [comment]
+1 for full implementation.
The above workaround creates a lot of overhead:
1. need to create 2 series more: the first with values of 0 and the other with values of the line itself.
2. need to add a custom tool tip in order to avoid the [0, interval value] that is automatically added when you use interval.
3. customize the tooltip with html to make it look better.

For some reason I could not make the custom tool tip work so I stopped trying...

**Comment by dlaliberte** [comment]
@HarelM Thanks for the outline for how to use smoothed 'area' intervals.  I agree it is tedious at best.

**Comment by Al1mirage** [comment]
+1

**Comment by Ram-SR** [comment]
@HarelM Can we draw multiple lines with area by above steps that you said? If we can, please show me an example. I tried many times but failed

**Comment by HarelM** [comment]
Chart configuration code can be found here:
https://github.com/IsraelHikingMap/Site/blob/master/IsraelHiking.Web/sources/application/components/RouteStatisticsChartComponent.ts
Which is used by the following directive:
https://github.com/IsraelHikingMap/Site/blob/master/IsraelHiking.Web/sources/application/directives/GoogleChartDirective.ts
Example of how it looks can be found here:
http://israelhiking.osm.org.il
By clicking the right pencil then click on the map on two places to create a route and then click on the bottom right chart icon to see the height graph. hover it to see the custom tooltip.

The following is the relevant issue the we developed the custom tooltip.
https://github.com/IsraelHikingMap/Site/issues/318

Bottom line, it works but requires some effort.

**Comment by Rik44444** [comment]
is there any progress on this? smooth lines on my area charts would be much appreciated!

**Comment by HarelM** [comment]
I moved to d3 a long time ago. I never looked back.

**Comment by Rik44444** [comment]
> 
> 
> I moved to d3 a long time ago. I never looked back.

what is d3? I use the Visualizer plugin for Wordpress, my options are Google Charts, DataTable or ChartJS
